### DISCUSSION TITLE
Curl Does v1/vector/collections/create not support creating additional fields?

### DISCUSSION THREAD

**Discussion by 490626721** [milvus-io/milvus#26345] [Q&A and General discussion]
According to the document, it seems that only primaryField and vectorField can be created. What should I do if I want to create additional fields?

**Comment by yhmo** [comment]
Milvus doesn't support adding/deleting fields from an existing collection. Sometimes we can use dynamic schema to add "un-defined fields": https://milvus.io/docs/dynamic_schema.md
